docs/data/joy/customization/right-to-left/right-to-left.md

```diff
@@ -0,0 +1,130 @@
+# Right-to-left language support
+
+<p class="description">Learn how to implement right-to-left (RTL) text with Joy UI to support languages such as Arabic, Persian, and Hebrew.</p>
+
+This guide outlines the three steps necessary to change the direction of text-based components in Joy UI to support RTL languages, as shown in the demo below:
+
+{{"demo": "RTLDemoJoy.js"}}
+
+## 1. Set the HTML direction
```

Comment:
As I'm reading over this again, I think we need to clarify that this is an "either/or" section:

```suggestion
## 1. Set the HTML direction

You can set the text direction either globally (across the entire app) or locally (specific to individual components), depending on your use case.
```